app/Keycloak/KeycloakServiceProvider.php

```diff
@@ -82,5 +84,6 @@ private function bootstrapEventHandling(): void
 
         Event::listen(IntegrationCreated::class, [CreateClients::class, 'handle']);
         Event::listen(IntegrationUpdated::class, [UpdateClients::class, 'handle']);
+        Event::listen(IntegrationBlocked::class, [BlockClients::class, 'handle']);
```

Comment:
Should this not be called `DisableClients`? Like you created `DisableClient` and `EnableClient`